deps/rabbit/src/rabbit_ff_registry.erl

```diff
@@ -178,7 +192,12 @@ is_enabled(FeatureName) ->
 %%   source code.
 
 is_registry_initialized() ->
-    always_return_false().
+    case inventory() of
+        init_required ->
+            false;
+        _Inventory ->
+            true
+    end.
```

Comment:
This could be simplified with:
```erlang
inventory() =/= init_required.
```

But perhaps you prefer to keep the same logic as other functions?

I'm fine with either solutions.